<launch>

	<node pkg="roomba_500driver_meiji" name="roomba_driver" type="main500"/>

	<node pkg="hokuyo_node" name="hokuyo_node" type="hokuyo_node"/>
		<param name="port" value="/dev/ttyACM0"/>

	<node pkg="roomba_teleop_meiji" name="electric_joystick_drive" type="electric_joystick_drive"/>

	<node pkg="joy" name="joy_node" type="joy_node"/>
		<param name="dev" value="/dev/input/js0"/>
	

</launch>
